ocs_ci/deployment/deployment.py

```diff
@@ -4234,7 +4252,12 @@ def deploy(self):
         odf_running_version = version.get_semantic_ocs_version_from_config()
         if odf_running_version >= version.VERSION_4_19:
             # create service exporter
-            create_service_exporter()
+            for cluster in get_non_acm_cluster_config():
+                index = cluster.MULTICLUSTER["multicluster_index"]
+                config.switch_ctx(index)
+                if get_provider_service_type() != "NodePort":
+                    create_service_exporter()
```

Comment:
This function will skip creation of serviceexport if platform is hci_baremetal or hci_vsphere. If this is handled here, plese update the function. The function also uses get_non_acm_cluster_config  function and do  not check client clusters.

Reply:
let me add confiton to skip for service_exporter